src/api/app/models/local_build_result/for_package.rb

```diff
@@ -33,8 +33,12 @@ def local_build_result(result, status)
                            code: status['code'], state: result['state'], details: status['details'])
     end
 
-    def excluded?(status)
-      !show_all && status == 'excluded'
+    def excluded_or_disabled?(status)
+      if show_all
+	return false
+      end
+	 
+      status == 'excluded' || status == 'disabled'
```

Comment:
what about `status.in?['excluded', 'disabled']`? :wink:

Reply:
Went with `['excluded', 'disabled'].include?(status)`